tests/modules/basics.c

```diff
@@ -893,6 +893,23 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     if (RedisModule_Init(ctx,"test",1,REDISMODULE_APIVER_1)
         == REDISMODULE_ERR) return REDISMODULE_ERR;
 
+    RedisModuleCallReply *reply = RedisModule_Call(ctx, "ping", "");
```

Comment:
let's add a comment as to what's the purpose of that PING

Reply:
Fixed